lib/jxl_extras.cmake

```diff
@@ -91,22 +91,17 @@ FetchContent_Declare(
 FetchContent_MakeAvailable(zlib libpng)
 endif()  # JPEGXL_EMSCRIPTEN
 
-find_package(ZLIB)  # dependency of PNG
-find_package(PNG)
-if(PNG_FOUND AND ZLIB_FOUND)
+if(NOT JPEGXL_BUNDLE_LIBPNG)
+  find_package(PNG)
```

Comment:
No need to find zlib? Or is that implied by png?

Reply:
Yes, the find module for PNG already looks for it.